### PR TITLE
Falcon 2075: Falcon HiveDR tasks do not report progress and can get killed

### PR DESCRIPTION
**Pull request by vrangan**

### CODE REVIEW

**Comment by PraveenAdlakha** [comment]
+1

File: addons/hivedr/src/main/java/org/apache/falcon/hive/mapreduce/CopyMapper.java [hunk 1]
```diff
@@ -54,15 +57,22 @@ protected void setup(Context context) throws IOException, InterruptedException {
...
     @Override
     protected void map(LongWritable key, Text value,
-                       Context context) throws IOException, InterruptedException {
+                       final Context context) throws IOException, InterruptedException {
         LOG.debug("Processing Event value: {}", value.toString());
-
+        timer = new ScheduledThreadPoolExecutor(1);
+        timer.scheduleAtFixedRate(new Runnable() {
+            public void run() {
+                System.out.println("Hive DR copy mapper progress heart beat");
```
Comment (bvellanki) [inline addons/hivedr/src/main/java/org/apache/falcon/hive/mapreduce/CopyMapper.java]: Please change this to LOG.debug

File: addons/hivedr/src/main/java/org/apache/falcon/hive/mapreduce/CopyReducer.java [hunk 2]
```diff
@@ -62,9 +65,18 @@ public int compare(ReplicationStatus r1, ReplicationStatus r2) {
...
     @Override
-    protected void reduce(Text key, Iterable<Text> values, Context context) throws IOException, InterruptedException {
+    protected void reduce(Text key, Iterable<Text> values, final Context context)
+            throws IOException, InterruptedException {
         List<ReplicationStatus> replStatusList = new ArrayList<ReplicationStatus>();
         ReplicationStatus rs;
+        timer = new ScheduledThreadPoolExecutor(1);
+        timer.scheduleAtFixedRate(new Runnable() {
+            public void run() {
+                System.out.println("Hive DR copy reducer progress heart beat");
```
Comment (bvellanki) [inline addons/hivedr/src/main/java/org/apache/falcon/hive/mapreduce/CopyReducer.java]: Please change this to LOG.debug

**Comment by bvellanki** [comment]
+1 apart from the minor nit about logging. @vrangan , thank you for the fix. This patch seems important for 0.10, so please update fix version in FALCON-2075 if you agree.

**Comment by vrangan** [comment]
Yes, please commit to 0.10

**Pull request closed without merging**